envConfig/preview1.env

```
@@ -1,7 +1,7 @@
 ## NB only variables prefixed with SIMORGH_ are available on both the server and client
 SIMORGH_BASE_URL=https://www.test.bbc.com
 SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=https://static.test.files.bbci.co.uk
-SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/ws/simorgh-preview-assets/public/
+SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/ws/simorgh-preview1-assets/public/
```

Would it be better to keep the name in line with the new simorgh preview name?
```suggestion
SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/ws/simorgh1-preview-assets/public/
```

---

Thanks, changed now for both